BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -32,20 +34,21 @@
     from edk2toollib.utility_functions import RunCmd
 
     #Tuple for (version, entrycount)
-    FORMAT_VERSION_1 = (1, 4)   #Version 1: #OVERRIDE : VERSION | PATH_TO_MODULE | HASH | YYYY-MM-DDThh-mm-ss
-    FORMAT_VERSION_2 = (2, 5)   #Version 2: #OVERRIDE : VERSION | PATH_TO_MODULE | HASH | YYYY-MM-DDThh-mm-ss | GIT_COMMIT
-    FORMAT_VERSIONS = [FORMAT_VERSION_1, FORMAT_VERSION_2]
-
+    OVERRIDE_FORMAT_VERSION_1 = (1, 4)   #Version 1: #OVERRIDE : VERSION | PATH_TO_MODULE | HASH | YYYY-MM-DDThh-mm-ss
+    OVERRIDE_FORMAT_VERSION_2 = (2, 5)   #Version 2: #OVERRIDE : VERSION | PATH_TO_MODULE | HASH | YYYY-MM-DDThh-mm-ss | GIT_COMMIT
+    DEPRECATION_FORMAT_VERSION_1 = (1, 3)   # Version 1: # DEPRECATED : VERSION | PATH_TO_NEW_MODULE_TO_USE | YYYY-MM-DDThh-mm-ss
+    FORMAT_VERSIONS = [OVERRIDE_FORMAT_VERSION_1, OVERRIDE_FORMAT_VERSION_2, DEPRECATION_FORMAT_VERSION_1]
 
     class OverrideValidation(IUefiBuildPlugin):
 
         class OverrideResult(object):
-            OR_ALL_GOOD             = 0
-            OR_FILE_CHANGE          = 1
-            OR_VER_UNRECOG          = 2
-            OR_INVALID_FORMAT       = 3
-            OR_DSC_INF_NOT_FOUND    = 4
-            OR_TARGET_INF_NOT_FOUND = 5
+            OR_ALL_GOOD               = 0
+            OR_FILE_CHANGE            = 1
+            OR_VER_UNRECOG            = 2
+            OR_INVALID_FORMAT         = 3
+            OR_DSC_INF_NOT_FOUND      = 4
+            OR_TARGET_INF_NOT_FOUND   = 5
+            OR_DEPRECATED_MODULE_USED = 6

```

Comment:
This is not an "OR_" define anymore, right? Should it be "DEC_" as it stands for deprecation?